### PR TITLE
Cindy/change covers

### PR DESCRIPTION
**Pull request by sweetpea22**
hey @lastperson! 
I only changed the title to Delv. I thought it would be okay to keep mentions of element finance in the audit body text. Let me know if you want me to remove all mentions of their old name.

### CODE REVIEW

**Comment by lastperson** [comment]
@sweetpea22 could the body be updated as well? Here is what they say:
The product is Hyperdrive, and the company is DELV. Element Finance shouldn’t be referred to anymore.

**Review by lastperson — CHANGES_REQUESTED** [review]
They've asked to change Element Finance to DELV everywhere.

File: index.json [hunk 1]
```diff
@@ -126,6 +126,22 @@
...
+    "name": "Hashnote Vaults",
+    "date": "02/2023",
+    "desc": "A review of the vaults and auction contracts used with Hashnote's options protocol ecosystem.",
+    "auditors": ["Anderson Lee", "Tanya Bushenyova", "Oleksii Matiiasevych"],
+    "path": "hashnote-vaults-02-2023",
+    "sourceCodeUrl": "",
+    "commit": "48ce04a3a43ec1dfbb335b6d4dd06aba3160c02f"
+  },
+  {
+    "name": "Grappa Finance",
```
Comment (lastperson) [inline index.json:138]: Something broke here.

Comment (sweetpea22) [reply index.json:138]: hm ok

**Review by lastperson — APPROVED** [review]

**Review by lastperson — CHANGES_REQUESTED** [review]
index.json need to reflect paths change.

**Review by lastperson — APPROVED** [review]

**Pull request merged**